styleguide/derek/pattern-components/leads/_lead.scss

```diff
@@ -1,5 +1,10 @@
 .lead-bodyWrapper {
   margin-bottom: 14px;
+
+  a {
+    color: $rackspace-red;
+    text-decoration: underline;
+  }
```

Comment:
If this is the route we're taking for this, it might make sense to just modify the `.lead-bodyWrapper > a` below to just be `.lead-bodyWrapper a` (i.e. change from the child combinator to the descendent combinator). 

The `standard-link` mixin has all the behavior we want anyway.  Apart from that, I this should be good (pending a thorough site review).

Reply:
I thought better not to jack with existing rules, but aye.

Reply:
In this case, it wouldn't actually modify the existing rule, other than making it more permissive, which has the exact same effect as the code you've provided above (just with a few other standard link caveats).